README.md

```diff
@@ -34,13 +34,13 @@ New Relic has two kinds of integrations:
 `newrelic-integration-e2e-action` can be installed as binary by using the following Go tool command:
 
 ```shell
-go install github.com/newrelic/newrelic-integration-e2e-action/newrelic-integration-e2e@latest
+go install github.com/newrelic/newrelic-integration-e2e@latest
```

Comment:
this new path looks wrong, missing the action suffix

Reply:
Good catch! Updated here c05bacc520c227b837457b28c99f3d6decba2ae6